### PR TITLE
Added prod tag to build production objects only

### PR DESCRIPTION
**Pull request by jimoibm**
@kadler 
I added a tag into Makefile.in so that I can get a clean environment to build for PTFs. Please help take a look. Thanks!

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -56,6 +58,11 @@ build-cgi: $(LIBRARY).lib xmlcgi.pgm
 
 build-test: $(TEST_PGMS) $(TEST_SRVPGMS)
 
+prod-clean:
+	rm -f *.lib *.pgm *.srvpgm *.module *.sqlinst *.srcpf
```
Comment (kadler) [inline Makefile.in]:
```suggestion
	rm -f *.module *.srcpf
```

Since you're only removing the file and module objects

Comment (jimoibm) [reply Makefile.in]: @kadler I don't why last two lines were lost in the compare. I  I want to only leave 5 pgm/svrpgm in library QXMLSERV. This is my changes:
```
prod: build-main build-procedure build-cgi prod-clean

prod-clean:
        rm -f *.lib *.pgm *.srvpgm *.module *.sqlinst *.srcpf
        rm -rf /qsys.lib/$(LIBRARY).lib/*.FILE
        rm -rf /qsys.lib/$(LIBRARY).lib/*.MODULE

```
Could you add that in the Makefile.in if it looks good to you? Thanks!

Comment (kadler) [reply Makefile.in]: They're there in the compare, I _suggested_ the above changes in my comment. There's no need to remove the `*.lib *.pgm *.srvpgm *.sqlinst` files if you're not removing those objects from the library.

Comment (kadler) [reply Makefile.in]: FYI: https://github.blog/changelog/2018-10-16-suggested-changes/

**Review by kadler — COMMENTED** [review]
The changes in Makefile.in look fine, but there seems to be a lot of unrelated changes in plugconf.rpgle.in and others.

**Comment by jimoibm** [comment]
@kadler I synced with upstream. There is only Makefile.in in the changed files. Not sure if this time it is correct. I don't know how to remove previous commits.

**Comment by kadler** [comment]
That's ok, they should go away when it gets squashed.

**Pull request merged**